www/docs/en/dev/guide/platforms/ios/webview.md

```diff
@@ -220,5 +222,16 @@ After using either of these two methods, continue with the **"Using CDVViewContr
     */
     viewController.wwwFolderName = @"myfolder";
     viewController.startPage = @"mypage.html"
-    ```
+    ```
+## Adding Plugins files used in you cordova project
```

Comment:
I'm not sure I agree with this section. This assumes that they already have a Cordova app. This whole section is for adding Cordova to an existing Xcode project, without needing cordova-cli and its tools. Granted a Cordova app without plugins is useless, but there is no workflow on how to do that, from start to finish. PhoneGap itself has instructions on how to do that: http://docs.phonegap.com/tutorials/develop/1-embed-webview/ios/ but that's packaging up the core plugins into Cocoapods (with a fixed config.xml). 

A generalized plugin add solution for an Embedded Webview workflow has not been implemented yet although we have discussed it (with no satisfactory conclusion).